string object_label
---
int32[] object_info
int32[] image_size